specs/core/0_beacon-chain.md

```diff
@@ -440,6 +444,25 @@ The following data structures are defined as [SimpleSerialize (SSZ)](https://git
 }
 ```
 
+##### `Transfer`
+
+```python
+{
+    # Sending from
+    'sender': 'uint64',
+    # Sending to
+    'to': 'uint64',
```

Comment:
Question to all eth1.0 and eth2.0 devs: do we want to keep using `to` in eth2.0...?

Reply:
I changed `sender` to `from` to be consistent with `to`. (We could also use `sender`/`recipient` but that's a bit more verbose.)